docs/users/favorite-campaigns.md

```diff
@@ -0,0 +1,15 @@
+---
+id: favorite-campaigns
+title: "Create a list of favorite campaigns"
+sidebar_label: "Create a list of favorite campaigns"
+---
+
+You can track the campaigns you want to work on by favoriting campaigns.
+
+To add a campaign as a favorite, go to the [campaigns list]({{appURL}}/campaigns) and select a campaign.
+
+Next click the **Add Favorite** button.
+
+![Add favorite campaign button](/docs/img/add-favorite-campaign.png)
+
+To remove the campaign from your favorites list, navigate back to that campaign and click the **Remove Favorite** button.
```

Comment:
This is a non-blocking comment, but it might be nice to have something to show that the Remove Favorite button replaces the Add Favorite button on the page so that people know where to find it